app/services/darken_service.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# Service for darkening an object.
+# This service removes files from the stacks filesystem in order to make an objct dark.
+class DarkenService
+  def self.darken(cocina_object)
+    new(cocina_object).darken
+  end
+
+  def initialize(cocina_object)
+    @stacks_druid_path = DruidTools::PurlDruid.new(cocina_object.externalIdentifier, Settings.filesystems.stacks_root).pathname
+  end
+
+  # Remove the object from the stacks filesystem and clean up any empty directories in the druid tree.
+  def darken
```

Comment:
Minor suggestion. For a service that only has a single method, name that method `call`.